### PR TITLE
35 f236 xus intro msg

### PR DESCRIPTION
**Pull request by kekuso**
- add implementation and tests for F236 XUS INTRO MSG

### CODE REVIEW

File: prototypes/files/f236_xusIntroMsg [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        processRPCInvocation(rpcArgs) {
+            // Any argument passed into function will throw an error
+            paramErrorHandler(rpcArgs);
+
+            // the kernel system parameters file is the only file in 8989.3
+            const introMsg = [];
+            introMsg.push(VDM.describe('8989_3-1').intro_message);
+
+            // return value must be an array
+            return introMsg;
```
Comment (mfuroyama) [inline prototypes/files/f236_xusIntroMsg]: @kekuso, let's talk about this tomorrow...

Comment (kekuso) [inline prototypes/files/f236_xusIntroMsg]: @mfuroyama will concatenating fix the issue?

**Comment by mfuroyama** [comment]
@kekuso Actually, I think you might have to go the other way, that is, return the results of `intro_message.split('\n')` instead of concatenating `intro_message` into the result.

I know it's hard to judge with a single line intro message, but the VDM model object will always return `intro_message` as a single string separated with by newlines ([it seems like it does that for all WORD PROCESSING field types fmType === 5](https://github.com/vistadataproject/VDM/blob/master/prototypes/fmql.js#L104))

**Comment by kekuso** [comment]
ohh interesting, ok I'll refactor the code thanks!

**Pull request merged**